package.json

```diff
@@ -17,9 +17,10 @@
     "move-node": "bin/move-node.js",
     "move-shard": "bin/move-shard.js",
     "move-shards": "bin/move-shards.js",
-    "remove-node": "bin/remove-node.js",
+    "remove-node": "bin/remove-node.js",   
```

Comment:
```suggestion
    "remove-node": "bin/remove-node.js",
```

Reply:
UGH I pay this price when  I resolve conflicts in GitHub UI...